paddleocr.py

```diff
@@ -685,15 +684,30 @@ def ocr(
         """
         OCR with PaddleOCR
 
-        args:
-            img: img for OCR, support ndarray, img_path and list or ndarray
-            det: use text detection or not. If False, only rec will be exec. Default is True
-            rec: use text recognition or not. If False, only det will be exec. Default is True
-            cls: use angle classifier or not. Default is True. If True, the text with rotation of 180 degrees can be recognized. If no text is rotated by 180 degrees, use cls=False to get better performance. Text with rotation of 90 or 270 degrees can be recognized even if cls=False.
-            bin: binarize image to black and white. Default is False.
-            inv: invert image colors. Default is False.
-            alpha_color: set RGB color Tuple for transparent parts replacement. Default is pure white.
-            slice: use sliding window inference for large images, det and rec must be True. Requires int values for slice["horizontal_stride"], slice["vertical_stride"], slice["merge_x_thres"], slice["merge_y_thres] (See doc/doc_en/slice_en.md). Default is {}.
+        Args:
+            img: Image for OCR. It can be an ndarray, img_path, or a list of ndarrays.
+            det: Use text detection or not. If False, only text recognition will be executed. Default is True.
+            rec: Use text recognition or not. If False, only text detection will be executed. Default is True.
+            cls: Use angle classifier or not. Default is True. If True, the text with a rotation of 180 degrees can be recognized. If no text is rotated by 180 degrees, use cls=False to get better performance. Text with a rotation of 90 or 270 degrees can be recognized even if cls=False.
```

Comment:
```suggestion
            cls: Use angle classifier or not. Default is True. If True, the text with a rotation of 180 degrees can be recognized. If no text is rotated by 180 degrees, use cls=False to get better performance.
```

If you want to support more angles, you may need to modify the algorithm.

<img width="172" alt="Snipaste_2024-07-11_20-16-53_0" src="https://github.com/PaddlePaddle/PaddleOCR/assets/17264618/2c0bf9de-6825-4a91-8022-986965d30f19">

```
[2024/07/11 20:17:52] ppocr INFO: [[[175.0, 23.0], [219.0, 22.0], [225.0, 252.0], [181.0, 253.0]], ('m ', 0.5273141264915466)]
[2024/07/11 20:17:52] ppocr INFO: [[[233.0, 22.0], [270.0, 22.0], [270.0, 238.0], [233.0, 238.0]], ('  ', 0.5910992622375488)]
[2024/07/11 20:17:52] ppocr INFO: [[[20.0, 34.0], [58.0, 34.0], [58.0, 94.0], [20.0, 94.0]], ('77', 0.9802588820457458)]
[2024/07/11 20:17:52] ppocr INFO: [[[15.0, 258.0], [62.0, 258.0], [67.0, 515.0], [20.0, 515.0]], ('   ss', 0.637114942073822)]
[2024/07/11 20:17:52] ppocr INFO: [[[182.0, 519.0], [218.0, 519.0], [215.0, 757.0], [179.0, 757.0]], ('  ', 0.6727192401885986)]
[2024/07/11 20:17:52] ppocr INFO: [[[233.0, 518.0], [273.0, 518.0], [271.0, 770.0], [231.0, 770.0]], ('S  ', 0.5468132495880127)]
[2024/07/11 20:17:52] ppocr INFO: [[[72.0, 622.0], [109.0, 622.0], [109.0, 769.0], [72.0, 769.0]], (' ', 0.5138227939605713)]
[2024/07/11 20:17:52] ppocr INFO: [[[123.0, 642.0], [169.0, 645.0], [164.0, 768.0], [118.0, 765.0]], ('‘LN-', 0.6751519441604614)]
[2024/07/11 20:17:52] ppocr INFO: [[[30.0, 781.0], [52.0, 781.0], [52.0, 822.0], [30.0, 822.0]], ('n0', 0.8192383050918579)]
[2024/07/11 20:17:52] ppocr INFO: [[[185.0, 777.0], [215.0, 777.0], [215.0, 918.0], [185.0, 918.0]], ('ss', 0.5055781006813049)]
[2024/07/11 20:17:52] ppocr INFO: [[[135.0, 1213.0], [159.0, 1213.0], [159.0, 1271.0], [135.0, 1271.0]], (' uo', 0.8076138496398926)]
[2024/07/11 20:17:52] ppocr INFO: [[[189.0, 1230.0], [211.0, 1230.0], [211.0, 1281.0], [189.0, 1281.0]], ('SIS', 0.8517476916313171)]
[2024/07/11 20:17:52] ppocr INFO: [[[180.0, 1297.0], [217.0, 1297.0], [217.0, 1546.0], [180.0, 1546.0]], (' se s', 0.5038232207298279)]
```